<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogProjectSettings</class>
 <widget class="QDialog" name="DialogProjectSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>478</width>
    <height>442</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>项目设置</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="1">
    <widget class="LatticeShow" name="widget" native="true">
     <property name="minimumSize">
      <size>
       <width>115</width>
       <height>115</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>115</width>
       <height>115</height>
      </size>
     </property>
     <property name="cursor">
      <cursorShape>ArrowCursor</cursorShape>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_4">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>类型定义</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0" colspan="2">
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="11" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_7">
       <property name="maximumSize">
        <size>
         <width>64</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>格式：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_Format">
       <property name="font">
        <font>
         <family>Microsoft YaHei</family>
        </font>
       </property>
       <item>
        <property name="text">
         <string>C文件</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>BIN文件</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="Line" name="line_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="10" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_8">
       <property name="minimumSize">
        <size>
         <width>64</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>64</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>路径：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_Output">
       <property name="font">
        <font>
         <family>Microsoft YaHei</family>
        </font>
       </property>
       <property name="placeholderText">
        <string>./</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="label_5">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>输出</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="12" column="1">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="label_6">
       <property name="maximumSize">
        <size>
         <width>64</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>常量修饰词：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_Const">
       <property name="font">
        <font>
         <family>Microsoft YaHei</family>
        </font>
       </property>
       <item>
        <property name="text">
         <string>const</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>code</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item row="7" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QLabel" name="label_10">
       <property name="maximumSize">
        <size>
         <width>64</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>图片尺寸:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_SizeTypedef">
       <property name="font">
        <font>
         <family>Microsoft YaHei</family>
        </font>
       </property>
       <item>
        <property name="text">
         <string>unsigned char</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>unsigned short int</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>unsigned int</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>char</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>short int</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>int</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>取模设置</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <layout class="QVBoxLayout" name="verticalLayout" stretch="1,0,0,8">
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>64</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>取模走向：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButton_LSB">
         <property name="font">
          <font>
           <family>Microsoft YaHei</family>
          </font>
         </property>
         <property name="text">
          <string>逆向(LSB)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButton_MSB">
         <property name="font">
          <font>
           <family>Microsoft YaHei</family>
          </font>
         </property>
         <property name="text">
          <string>顺向(MSB)</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="minimumSize">
          <size>
           <width>64</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>取模方式：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButton_ZL">
         <property name="font">
          <font>
           <family>Microsoft YaHei</family>
          </font>
         </property>
         <property name="text">
          <string>逐列</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButton_ZH">
         <property name="font">
          <font>
           <family>Microsoft YaHei</family>
          </font>
         </property>
         <property name="text">
          <string>逐行</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButton_LH">
         <property name="font">
          <font>
           <family>Microsoft YaHei</family>
          </font>
         </property>
         <property name="text">
          <string>列行</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButton_HL">
         <property name="font">
          <font>
           <family>Microsoft YaHei</family>
          </font>
         </property>
         <property name="text">
          <string>行列</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="6" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QLabel" name="label_9">
       <property name="maximumSize">
        <size>
         <width>64</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>图片位置：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_PosTypedef">
       <property name="font">
        <font>
         <family>Microsoft YaHei</family>
        </font>
       </property>
       <item>
        <property name="text">
         <string>unsigned char</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>unsigned short int</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>unsigned int</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>char</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>short int</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>int</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LatticeShow</class>
   <extends>QWidget</extends>
   <header location="global">custom/latticeshow.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogProjectSettings</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogProjectSettings</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
